### PR TITLE
chore: add  __str__ methods

### PR DESCRIPTION
**Pull request by shashank-k-y**
#59

### CODE REVIEW

File: bot/models/github/event.py [hunk 1]
```diff
@@ -44,3 +44,6 @@ def __init__(self, event_type: EventType, repo: Repository, **kwargs):
         self.commits: list[Commit] | None = kwargs.get("commits", None)
         self.links: list[Link] | None = kwargs.get("links", None)
         self.reviewers: list[User] | None = kwargs.get("reviewers", None)
+
+    def __str__(self):
+        return f"<{self.type}|{self.repo}>"
```
Comment (BURG3R5) [inline bot/models/github/event.py:49]: Here it'd be better if you return a string that looks like a named tuple that contain _all_ the data in the object. Something like:
```py
return f"""(
type={self.type},
repo={self.repo},
users={[str(r) for r in self.reviewers]},
...  # Remaining attributes
)"""
```

**Review by BURG3R5 — CHANGES_REQUESTED** [review]
Everything looks great, except the `GitHubEvent` model. See comment for more info.

File: bot/models/github/event.py (hunk 1 above)
Comment (shashank-k-y) [reply bot/models/github/event.py]: [updated string](https://github.com/BURG3R5/github-slack-bot/pull/62/commits/5d541b9a671d5b2c7813cbf2235ebb721657eb8a)

**Review by BURG3R5 — APPROVED** [review]

**Comment by BURG3R5** [comment]
@all-contributors please add @shashank-k-y for code

**Pull request merged**